invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/landing_page/ShareOptions/AccessLinks/AccessDropdown.js

```diff
@@ -83,4 +84,6 @@ AccessDropdown.propTypes = {
   result: PropTypes.object.isRequired,
   dropdownOptions: PropTypes.array.isRequired,
   updateEndpoint: PropTypes.string.isRequired,
+  entityType: PropTypes.string.isRequired,
+  updatedCallback: PropTypes.func.isRequired,
```

Comment:
can I suggest a rename for something like:
```suggestion
  onItemUpdated: PropTypes.func.isRequired,
```
?